src/components/ParentCryptoCurrencyIcon.js

```diff
@@ -1,21 +1,21 @@
 // @flow
 import React, { PureComponent } from 'react'
 import styled from 'styled-components'
-import get from "lodash/get"
+import get from 'lodash/get'
 import type { Currency } from '@ledgerhq/live-common/lib/types'
-import CryptoCurrencyIcon from "./CryptoCurrencyIcon"
+import CryptoCurrencyIcon from './CryptoCurrencyIcon'
```

Comment:
this is in fact correct changes.

@IAmMorrow please make sure to configure your editor to prettier at save.